package.json

```diff
@@ -11,6 +11,7 @@
   ],
   "scripts": {
     "test": "npm run lint -s",
+    "tap-test": "prova -t provify ./tests/**/**/*.spec.js ./tests/**/**/**/*.spec.js ./tests/**/*.spec.js -b",
```

Comment:
I have an implementation using testling that we might want to explore.  It is headless and has better stack traces